cmake/compiler_support.cmake

```diff
@@ -302,8 +302,17 @@ function(create_omr_compiler_library)
       list(REMOVE_ITEM CORE_COMPILER_OBJECTS ${abs_filename})
    endforeach()
 
+   # Convert pasm files to asm files: run this before masm2gas to ensure 
+   # asm files subsequently get picked up. 
+   pasm2asm_files    (CORE_COMPILER_OBJECTS ${COMPILER_NAME} ${CORE_COMPILER_OBJECTS})
```

Comment:
Removed; they were there to align with the below to allow me to visually ensure the parallel with masm2gas.